internal.lua

```diff
@@ -3,19 +3,26 @@ function areas:player_exists(name)
 	return minetest.get_auth_handler().get_auth(name) ~= nil
 end
 
+local safe_file_write = minetest.safe_file_write
+if safe_file_write == nil then
+	function safe_file_write(path, content)
```

Comment:
nothing about this function is safe so the naming is misleading
(I know it's just a fallback)